### ISSUE TITLE
parallel mode incorrectly skips some tests

### ISSUE THREAD

**Issue by sebns** [hrcorval/behavex#84]
**Describe the bug**
`behavex` in parallel mode incorrectly skips some tests.
This happens when the `--tags` argument refers to a tag attached to a Scenario Outline Example.

**To Reproduce**
With this feature file:

```
Feature:

    Scenario Outline:
        Given the initial value is <initial>
        When the value is incremented by <increment>
        Then the value is <result>

        @foo
        Examples:
            | initial | increment | result |
            |       4 |       3   |      7 |
            |      14 |      13   |     27 |
```

Commands executed:
```sh
$ behave --tags=foo
  # displays:  2 scenarios passed...
$ behavex --tags=foo
  # displays:  2 scenarios passed...
$ behavex --tags=foo --parallel-processes=2 --parallel-scheme=feature
  # no tests were performed, but expected 2 scenarios
```

**Expected behavior**
behavex in parallel should run the tests tagged `foo` (exactly as both behavex serial and behave do).

**Desktop:**
- Docker image `python:3.11` (i.e. Debian)
- behavex==2.0.1 and behave==1.2.6

**Comment by anibalinn** [comment]
Fixed in upcoming release branch [release_2.0.2](https://github.com/hrcorval/behavex/tree/release_2.0.2)
Thanks a lot!

**Comment by anibalinn** [comment]
Fix released in version 3.0.0: https://pypi.org/project/behavex/3.0.0/
branch [release_3.0.0](https://github.com/hrcorval/behavex/tree/release_3.0.0)
Thanks!
